sdks/python/apache_beam/utils/options.py

```diff
@@ -488,6 +488,24 @@ def _add_argparse_args(cls, parser):
          '(--staging_location option) and the workers will install them in '
          'same order they were specified on the command line.'))
 
+
+class TestOptions(PipelineOptions):
+
+  @classmethod
+  def _add_argparse_args(cls, parser):
+    # Options for e2e test pipeline.
+    parser.add_argument(
+        '--on_success_matcher',
```

Comment:
`onCreateMatcher` in Java is not used in ITs currently. I prefer only keep `onSuccessMatcher` unless there is a need for `onCreateMatcher` for Python.